### PR TITLE
Clean up aria labels

### PR DESCRIPTION
**Pull request by Jade-Codes**
### What does this PR do?
Cleans up some of the aria labels to make them more inline with accessibility guidelines

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: client/src/pages/spectrogram/components/annotation/AnnotationViewer.tsx [hunk 1]
```diff
@@ -185,7 +185,7 @@ const AnnotationViewer = ({
 
   return (
     <Layer>
-      {/* Button to add a new annotation */}
```
Comment (hannahlspencer) [inline client/src/pages/spectrogram/components/annotation/AnnotationViewer.tsx:188]: Presuming you didn't mean to remove the Button here

Comment (Jade-Codes) [reply client/src/pages/spectrogram/components/annotation/AnnotationViewer.tsx]: Good spot! I did not

Comment (Jade-Codes) [reply client/src/pages/spectrogram/components/annotation/AnnotationViewer.tsx]: Added that back in :)

**Review by hannahlspencer — APPROVED** [review]
LGTM

**Pull request merged**